### PR TITLE
Simplify example to use webpack

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -7,17 +7,24 @@
 |----------------------------------------------------------------------------*/
 'use strict';
```
Comment (sccolbert) [inline index.ts]: Why change this to a `.ts` file instead of leaving it `.js`?

Comment (blink1073) [inline index.ts]: So it would pull from the `.ts` files

File: index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <link rel="stylesheet" href="theme/index.css">
-  <script src="node_modules/steal/steal.js" main="index"></script>
+  <script src="bundle.js" main="index"></script>
```
Comment (sccolbert) [inline index.html]: does the `main` attribute have any meaning anymore?

Comment (blink1073) [inline index.html]: Nope, removed.

**Comment by sccolbert** [comment]
:+1:

**Pull request merged**